src/components/FormContainer.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+
+function FormContainer({link, title, form, accountMessage, alternateTitle}) {
+  return (
+
+    <div className="container h-100">
+        <div className="d-flex justify-content-center h-100">
+            <div className="user-card">
+                <div className="d-flex justify-content-center">
+                    <h3 className="form-title">{title}</h3>
+                </div>
+
+                <div className="d-flex justify-content-center form-container">
+                    {form()}
+                </div>
+
+                <div className="mt-4">
+                    <div className="d-flex justify-content-center links">
+                        {accountMessage} <Link to={link} className="ml-2">{alternateTitle}</Link>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+  )
+}
```

Comment:
Use all traditional functions are all arrow functions.

Use children prop to render the form component.
```suggestion
function FormContainer({link, title, form, accountMessage, alternateTitle, children}) {
  return (

    <div className="container h-100">
        <div className="d-flex justify-content-center h-100">
            <div className="user-card">
                <div className="d-flex justify-content-center">
                    <h3 className="form-title">{title}</h3>
                </div>

                <div className="d-flex justify-content-center form-container">
                    {children}
                </div>

                <div className="mt-4">
                    <div className="d-flex justify-content-center links">
                        {accountMessage} <Link to={link} className="ml-2">{alternateTitle}</Link>
                    </div>
                </div>
            </div>
        </div>
    </div>
  )
}
```